src/web/index.html

```diff
@@ -1,14 +1,36 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="br">
 <head>
     <meta charset="UTF-8">
-    <title>Hello World</title>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <link rel="stylesheet" href="app/css/login.css">
```

Comment:
Você não pode pegar uma stylesheet no /app se você estiver na página de login (no index), pois o servidor só te deixa acessar a /app se vc estiver logado, troque o lugar para, por enquanto na mesma pasta que a index, depois colocamos uma subpasta pra ficar organizado
```suggestion
    <link rel="stylesheet" href="login.css">
```